modules/core/src/main/java/org/apache/ignite/internal/management/cdc/CdcDeleteLostSegmentsTask.java

```diff
@@ -142,7 +141,7 @@ protected CdcDeleteLostSegmentsJob(CdcDeleteLostSegmentLinksCommandArg arg, bool
                     Path stateDir = ft.walCdc().toPath().resolve(STATE_DIR);
 
                     if (stateDir.toFile().exists()) {
-                        File walState = stateDir.resolve(WAL_STATE_FILE_NAME).toFile();
+                        File walState = NodeFileTree.walStateFilePath(stateDir, false).toFile();
```

Comment:
This must be NOT static method - we have all information inside `NodeFileTree`, already.